CHANGELOG.md

```diff
@@ -3,17 +3,11 @@
 ## Unreleased
 
 ### Bug Fixes
-
+- Return Swift optionals from Objective C methods that can return nil
```

Comment:
this does not seem to fit our format. it should be under a section heading and also have a reference to the PR or issue

Reply:
Added PR link in 9c49343.